packages/deployments/contracts/src/cli/ownership/helpers/constants.ts

```diff
@@ -32,15 +32,15 @@ export const PROTOCOL_ADMINS: AddressBook = {
     1886350457: "0x95e687bA2d2A791B0246d66bC4fbdD0457dCab19", // polygon
     1634886255: "0xC1167F7209A7434252e427Ad96693ccf6FB2755F", // arbitrum one
     1818848877: "0x50d22a34183e7cD3164184011998a3D50303f75d", // linea
-    1835365481: "0xC6E8d50c86BbBD974228B38d02EBBDdF109e047d", // metis
-    1835101812: "0xeE26cb410E2afF7D8e342AA84746F3459eCd643f", // mantle
-    1887071085: "0x3FdcFaD98F37A7Cd713b03e9D01E2193Ea91234C", // polygonzk
-    1635148152: "0x63aFBf69628F09a2f7a44c0546b03e933ED86C01", // avalanche
-    1650553709: "0x5470aBdf216E557948aCaE671b29D3BCdE0D6F3c", // base
-    2053862243: "0xc0e2a1249c12f3d1B33bb3D319eBa0197dFB0641", // zksync-era
-    1836016741: "0x1222165e4087304c60fC76Ed47Ab2887eD238108", // mode
-    1935897199: "0xdD247760618092f0F769E10Ad9f5c3e5B67E545e", // scroll
-    2020368761: "0xade09131C6f43fe22C2CbABb759636C43cFc181e", // xlayer
+    1835365481: "0xa81a3e9b6fdd1c00785082eee42affe62d3f1217", // metis
+    1835101812: "0xa1bf64c4c185570bd79354296097edad915a45d3", // mantle
+    1887071085: "0xa1bf64c4c185570bd79354296097edad915a45d3", // polygonzk
+    1635148152: "0xa1bf64c4c185570bd79354296097edad915a45d3", // avalanche
+    1650553709: "0xa1bf64c4c185570bd79354296097edad915a45d3", // base
+    2053862243: "0x92290ba80ac3a6d6fd0a23e74de692d684435572", // zksync-era
+    1836016741: "0x739eea65c622bfa4a9e664ae37334a4e14c4b2dd", // mode
+    1935897199: "0x739eea65c622bfa4a9e664ae37334a4e14c4b2dd", // scroll
+    2020368761: "0xf7b522d25f50f86db273975c1a7f11a7d11db6f9", // xlayer
```

Comment:
Is this Ok? protocol admin for some chains are updated?

Reply:
yes